src/Pages/ProfilePage/ProfilePage.jsx

```diff
@@ -1,5 +1,70 @@
+import { useAuth, useModal, usePebbleNote } from "../../Context";
+import "./ProfilePage.css";
+import Button from "../../Components/UI/Button/Button";
+import { useNavigate } from "react-router-dom";
+
 const ProfilePage = () => {
-  return <div className="body-content">ProfilePage</div>;
+  const {
+    auth: {
+      user: { firstName, lastName, dp, email },
+    },
+    authDispatch,
+  } = useAuth();
+  const {
+    state: { deletedNotes, allNotes, archivedNotes },
+  } = usePebbleNote();
+  const { setError, setShowError } = useModal();
+  const navigate = useNavigate();
+
+  const logOutHandler = () => {
+    authDispatch({ type: "logout" });
+    setError("Your have been logged out");
```

Comment:
It is somewhat ambiguous that you are treating the case of logout as an error due to the variable names.

Reply:
The variable was created long ago, I will fix this.